spec.bs

```diff
@@ -1021,13 +1021,12 @@ update called "owners":
 
 # Permissions Policy integration # {#permissions-policy-integration}
 
-This specification defines two [=policy-controlled features=] identified by the string
+This specification defines two [=policy-controlled features=] identified by the strings
 "<code><dfn noexport>join-ad-interest-group</dfn></code>", and
 "<code><dfn noexport>run-ad-auction</dfn></code>". Their
-[=policy-controlled feature/default allowlists=] are `self`.
+[=policy-controlled feature/default allowlists=] are "`*`".
 
-Note: In the Chromium implementation the [=policy-controlled feature/default allowlists=] for both
-features are temporarily set to `*` to ease testing.
+Issue(WICG/turtledove#522): consider moving from "`*`" to "`self`".

```

Comment:
I'd change from "considering" to something stronger as this is part of the plan, otherwise LGTM.

Reply:
Cool, changed to "Move".